client/my-sites/plugins/main.jsx

```diff
@@ -440,6 +434,9 @@ export default connect(
 			canJetpackSiteUpdateFiles: siteId => canJetpackSiteUpdateFiles( state, siteId ),
 			isJetpackSite: siteId => isJetpackSite( state, siteId ),
 			wporgPlugins: state.plugins.wporg.items,
+			userCanManagePlugins: ( !! selectedSite
+				? canCurrentUser( state, selectedSite.ID, 'manage_options' )
```

Comment:
In this case we had the selected site but not the selected site id.  The siteId that you see is existing code that created function that receives siteId and passed it as a prop. There is no need to pass functions as props in this case and in the pull request where I remove sites-list from main I changed this to pass all the information needed instead of props functions. 
Anyway we don't have siteId variable that we can use do you think we should create one using getSelectedSiteId ?